aip/general/0180.md

```diff
@@ -139,6 +139,84 @@ code, as such changes will be seen as breaking by those users.
 whether a proposed change is likely to break users, and an expansive reading of
 this guidance could ostensibly prevent _any_ change (which is not the intent).
 
+#### Default values must not change
+
+Changing the default value is considered breaking and **must not** be done. The
+default behavior for a resource is determined by it's default values, and this
+**must not** change across minor versions.
+
+For example:
+
+```proto
+// A representation of an automobile
+message Automobile {
+    // The name of the automobile.
+    string name = 1;
+    // The number of wheels of the automobile.
+    // The default is set to 2 when no value is sent by the client
```

Comment:
That's not really setting a default, it's setting the number of wheels. How about:

```text
// If this is not set when the automobile is created, the field will be given a value of 2.
```